SPECS/openssl.spec

```diff
@@ -500,7 +506,7 @@ rm -rf $RPM_BUILD_ROOT/%{_libdir}/fips_premain.*
 rm -rf $RPM_BUILD_ROOT/%{_libdir}/fipscanister.*
 
 %clean
-[ "$RPM_BUILD_ROOT" != "/" ] && rm -rf $RPM_BUILD_ROOT
+ "$RPM_BUILD_ROOT" != "/" ] && rm -rf $RPM_BUILD_ROOT
```

Comment:
This removal looks suspicious :)
Otherwise LGTM

Reply:
And it is the same in the other PR

Reply:
Fixed